exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/model_management.ipynb

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
<p>I understand that <code>model_delete</code> is used for testing. Could you do it the same way as in the statement above? <code>%%sql --save udf_model_delete</code>. Ideally, the testing should not interfere with the demo. Doing it your way would certainly raise a question of why do I have to assign the output to a variable. The presence of the --save parameter is also not particularly clear, but at least it's the same everywhere and it can be explained in a comment, if not explained already.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/418/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Done